src/schema/user/__tests__/api.js

```diff
@@ -54,5 +128,10 @@ query {
 export {
     user,
```

Comment:
Please, include the new fields, in [`user`](https://github.com/panelinhadees/server/blob/57cfde1671d4d6d4275fc5e709a4f31b045dc431/src/schema/user/__tests__/api.js#L4-L20) query and change `expctedResults.js` with new informations.